app/controllers/tips_controller.rb

```diff
@@ -33,6 +33,8 @@ def update
   end
 
   def destroy
+    Tip.destroy(@tip)
```

Comment:
change to `@tip.destroy`
see http://apidock.com/rails/v2.3.8/ActiveRecord/Base/destroy